tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/vertx/ActiveMQVertxUnitTest.java

```diff
@@ -282,19 +280,7 @@ public void testIncomingEvents() throws Exception {
       assertEquals(aShort, recvShort);
 
       //send a JsonObject
-      String jsonObjectString = "{\n" +
-         "\"Image\": {\n" +
-         "\"Width\":  800,\n" +
-         "\"Height\": 600,\n" +
-         "\"Title\":  \"View from 15th Floor\",\n" +
-         "\"Thumbnail\": {\n" +
-         "\"Url\":    \"http://www.example.com/image/481989943\",\n" +
-         "\"Height\": 125,\n" +
-         "\"Width\":  100\n" +
-         "},\n" +
-         "\"IDs\": [116, 943, 234, 38793]\n" +
-         "}\n" +
-         "}";
+      String jsonObjectString = "{\n" + "\"Image\": {\n" + "\"Width\":  800,\n" + "\"Height\": 600,\n" + "\"Title\":  \"View from 15th Floor\",\n" + "\"Thumbnail\": {\n" + "\"Url\":    \"http://www.example.com/image/481989943\",\n" + "\"Height\": 125,\n" + "\"Width\":  100\n" + "},\n" + "\"IDs\": [116, 943, 234, 38793]\n" + "}\n" + "}";
```

Comment:
In my opinion, unwrapping these lines (and other like this) makes it harder to read/change.